src/test/js/self_registration_test.js

```diff
@@ -112,15 +114,18 @@ Scenario('@functional @selfregister I can self register', async (I) => {
     I.fillField('#password1', TestData.PASSWORD);
     I.fillField('#password2', TestData.PASSWORD);
     I.click('Continue');
-    I.waitForText('Account created', 20, 'h1');
+    I.wait(30);
+    I.waitForText('Account created', 40, 'h1');
     I.see('You can now sign in to your account.');
     I.amOnPage(loginPage);
     I.seeInCurrentUrl("state=selfreg");
+    I.wait(30);
```

Comment:
I don't think 30 secs wait required here as this scenario passed in the current preview build without this.